### PR TITLE
Rent endpoint

### PR DESCRIPTION
**Pull request by AleFalcon**
## Summary
* Creation route index
* Creation route create
* Authentication for actions

## Trello Card
https://trello.com/c/ieeXabbr/28-crear-endpoint-de-creacion-de-rent-e-index

### CODE REVIEW

File: app/controllers/api/v1/rent_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      include Wor::Paginate
+
+      before_action :authenticate_user!
+
+      def index
+        render_paginated Rent, each_serializer: RentSerializer
+      end
+
+      def create
+        @rent = Rent.new(current_user, rent_params)
```
Comment (nicozar95) [inline app/controllers/api/v1/rent_controller.rb]:
```suggestion
        @rent = Rent.new(rent_params.merge(user_id: current_user.id))
```
If your version did not work.
Also check this guides for potencial exploits: https://rails-sqli.org/

File: app/controllers/book_controller.rb [hunk 2]
```diff
@@ -1,19 +0,0 @@
-class BookController < ApiController
```
Comment (nicozar95) [inline app/controllers/book_controller.rb]: Why did you delete this controller?

File: app/models/user.rb [hunk 3]
```diff
@@ -5,7 +5,6 @@ class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
   # :recoverable, :rememberable, :trackable, :validatable
-  has_many :rent, dependent: :destroy
```
Comment (nicozar95) [inline app/models/user.rb]: Same as above, this relation is necesary

File: app/controllers/book_controller.rb (hunk 2 above)
Comment (AleFalcon) [reply app/controllers/book_controller.rb]: because this controller is in other folder

File: app/models/user.rb (hunk 3 above)
Comment (AleFalcon) [reply app/models/user.rb]: Thi relation is in line 9

File: app/controllers/api/v1/rent_controller.rb (hunk 1 above)
Comment (AleFalcon) [reply app/controllers/api/v1/rent_controller.rb]: Ready

Comment (AleFalcon) [reply app/controllers/api/v1/rent_controller.rb]: Ready

File: app/controllers/api/v1/rent_controller.rb [hunk 4]
```diff
@@ -0,0 +1,27 @@
+module Api
+  module V1
+    class RentController < ApiController
+      include Wor::Paginate
+      before_action :authenticate_user!
+
+      def index
+        render_paginated Rent, each_serializer: RentSerializer
```
Comment (nicozar95) [inline app/controllers/api/v1/rent_controller.rb]: This is ok, but we want to see the rents of the `current_user` we dont need to see all the rents in the application
```
def index
  render_paginated Rent.where(user_id: current_user.id), each_serializer: RentSerializer
end
```

Comment (AleFalcon) [reply app/controllers/api/v1/rent_controller.rb]: Ready

**Pull request merged**